docs/packages/dev-workflow.md

```diff
@@ -2,107 +2,175 @@
+      REGISTRY_PASSWORD="****" \
+      REGISTRY_SERVER="****" \
+      make package-bundle-push -f build-tooling.mk
    ```
 
-   PACKAGE_BUNDLES variable should be comma-separated values and must not contain spaces.
-   Example: PACKAGE_BUNDLES=featuregates,core-management-plugins
-
-7. Push repo bundle
+6. Push repo bundle [Optional]
 
    To push the generated repo bundle to an OCI registry, run:
 
    ```shell
-      PACKAGE_REPOSITORY=management OCI_REGISTRY=${REGISTRY} make push-package-repo-bundle
+      PACKAGE_REPOSITORY="management" \
+      OCI_REGISTRY="****" \
+      REPO_BUNDLE_VERSION="v1.0.0" \
+      REPO_BUNDLE_SUB_VERSION="0" \
+      REGISTRY_USERNAME="****" \
+      REGISTRY_PASSWORD="****" \
+      REGISTRY_SERVER="****" \
+      make repo-bundle-generate repo-bundle-push -f build-tooling.mk
    ```
 
+----
+
 Follow the below steps to test the artifacts that are generated in previous steps on your local cluster
 
-**Note**: A local cluster can be any k8s cluster with `kapp-controller` installed in it.
-You can also use [management cluster](https://github.com/vmware-tanzu/tanzu-framework/blob/main/cmd/cli/plugin/managementcluster/README.md)
-or [workload cluster](https://github.com/vmware-tanzu/tanzu-framework/blob/main/cmd/cli/plugin/cluster/README.md) which
-comes with `kapp-controller` already installed in it to test the artifacts.
+> **Note**: A local cluster can be any k8s cluster. There are 2 ways to install the packages
 
-We can use the `package` plugin to add the repo bundle to the cluster and to install the packages
+#### To install on a cluster without `kapp-controller` installed in it.
 
-1. Add package repo bundle to the cluster
+1. Download the package bundle. The image sha tag can be noted from the registry where package was published.
+   ```shell
+      imgpkg pull -b ${OCI_REGISTRY}/readiness@sha256:1fb9f9c6f0c6ba1f995440885b02806551a79d9cef5b9c7c3d6f53a586facddd -o readiness-pkg
+   ```
 
+1. Install the package
    ```shell
-      tanzu package repository add repo --url ${REPO_BUNDLE_URL} --namespace ${NAMESPACE}
+      ytt -f readiness-pkg/config/ | kbld -f - -f readiness-pkg/.imgpkg/images.yml | kubectl apply -f-
    ```
 
-2. Install the package
+#### To install on a cluster with `kapp-controller` installed in it.
+
+> For more details about these steps, see kapp [packaging tutorial](https://carvel.dev/kapp-controller/docs/v0.31.0/packaging-tutorial)
+
+1. Create a `PackageRepository` resource. The image sha tag can be noted from the registry where **repo bundle** was published.
+
+   Create `repo.yml`
+   ```yaml
+      ---
+      apiVersion: packaging.carvel.dev/v1alpha1
+      kind: PackageRepository
+      metadata:
+      name: tanzu-management-repo
+      spec:
+      fetch:
+         imgpkgBundle:
+            image: ${OCI_REGISTRY}/management@sha256:1fb9f9c6f0c6ba1f995440885b02806551a79d9cef5b9c7c3d6f53a586facddd
+   ```
+
+   ```bash
+      kapp deploy -a repo -f repo.yml -y
```

Comment:
nit: let's keep the extension consistent as `yaml`

Reply:
Updated